File: 오선민/programmers/P120891.py
'''
문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/120891
시간: 0.01ms
메모리: 10.1MB
PR 제목 : 날짜 사이트_명 문제_번호 사용_언어
'''

def solution(order):
    order_count = str(order).count('3') + str(order).count('6') + str(order).count('9')
    return order_count

File: 강재연/programmers/P120891.js
// 문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/120891
// 시간: 0.05 ms
// 메모리: 33.4 MB

function solution(order) {
    let answer = 0;
    [...String(order)].forEach((num)=>{
        if(num == '3' || num == '6' || num == '9') answer++;
    });
    return answer;
}
